### PR TITLE
Draft of 0.9.1 release announce

### PR DESCRIPTION
**Pull request by serge-sans-paille**
@jeanlaroche can you proof read that article?

### CODE REVIEW

**Comment by jeanlaroche** [comment]
Sure

File: content/2019-01-20-pythran-0.9.1.rst [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+important date, when Python 2 will no longer be officially supported.
+Following `the move <https://python3statement.org/>`_ Pythran will stop
+supporting Python2 by the end of the year. Of course, the last stable version
+supporting Python2 will still be available at that movment, but only Python3
+will receive updates. It's in one year, but you'll be warned!
+
+Reminder
+========
+
+Pythran is an ahead-of-time compiler for numeric kernel. The whole idea is that you extract the high-level kernel you wrote using NumPy calls and stuff into an independent module, then run
```
Comment (jeanlaroche) [inline content/2019-01-20-pythran-0.9.1.rst]: stuff it into

File: content/2019-01-20-pythran-0.9.1.rst [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+
+Changelog
+=========
+
+So what happened? From the changelog:
+
+Honor PYTHRANRC environment variable for config file lookup
+-----------------------------------------------------------
+
+Pythran is now sensible to the ``PYTHRANRC`` environment variable. You can use it to point to different configuration, say different compiler and/or compiler settings:
```
Comment (jeanlaroche) [inline content/2019-01-20-pythran-0.9.1.rst]: Pythran now pays attention to the .... (sensible means "raisonable")

File: content/2019-01-20-pythran-0.9.1.rst [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+
+Favor Python3 support in various places
+---------------------------------------
+
+Rember the Python3 statement from the beginning of this post?
+
+Fix Numpy.remainder implementation
+----------------------------------
+
+That was a funny one: ``std::remainder`` from C++ and ``numpy.remainder`` don't have the same semantic when dealing with negative numbers.
```
Comment (jeanlaroche) [inline content/2019-01-20-pythran-0.9.1.rst]: "Don't yield identical results with negative numbers" (I don't think it's a question of semantics). You should confirm that pythran now does what numpy does, not what std::remainder does...

**Comment by jeanlaroche** [comment]
That's all I found. Great job!

**Comment by serge-sans-paille** [comment]
Thanks @jeanlaroche o/

**Pull request merged**